nilearn/datasets.py

```diff
@@ -83,27 +83,32 @@ def _chunk_report_(bytes_so_far, total_size, t0):
     bytes_so_far: int
         Number of downloaded bytes
 
-    total_size: int, optional
-        Total size of the file. None is valid
+    total_size: int
+        Total size of the file.
 
-    t0: int, optional
+    t0: int
         The time in seconds (as returned by time.time()) at which the
-        download was started.
+        download was resumed / started.
+
+    initial_size: int
+        If resuming, indicate the initial size of the file.
+        If not resuming, set to zero.
     """
-    if total_size:
-        percent = float(bytes_so_far) / total_size
```

Comment:
Blew away the old algorithm (well, it's essentially the same, but easier to PEP8)